libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -713,6 +713,7 @@ singleton AP_EFI method get_state void EFI_State'Ref
 -- ----END EFI Library----
 
 include AP_Logger/AP_Logger.h
+singleton AP_Logger depends (HAL_LOGGING_FILESYSTEM_ENABLED == 1)
```

Comment:
This doesn't look right; you don't need filesystem support for logging - it can be accomplished via mavlink or dataflash.

There is a `HAL_LOGGING_ENABLED` which might be suitable?

Reply:
@peterbarker the specific functions it refers actually depend on HAL_LOGGING_FILESYSTEM_ENABLED  , HAL_LOGGING_ENABLED can be true (mavlink logging e.g ) the latter might not be.